### PR TITLE
Add Kipkluif

### PR DESCRIPTION
**Pull request by tmagicturtle**

### CODE REVIEW

File: data/mods/ssb/moves.js [hunk 1]
```diff
@@ -2137,6 +2137,79 @@ let BattleMovedex = {
...
+				// @ts-ignore
+				sourceBoosts[stat] = pokemon.boosts[stat];
+			}
+
+			pokemon.setBoost(targetBoosts);
+			target.setBoost(sourceBoosts);
+
+			this.add('-swapboost', pokemon, target, 'def, spd', '[from] move: Guard Swap');
+		},
+		effect: {
```
Comment (HoeenCoder) [inline data/mods/ssb/moves.js]: Since this is defined in the code for king's sheild I'm pretty sure this is not needed. Please test before changing.

**Review by HoeenCoder — COMMENTED** [review]
Animation missing.

**Comment by HoeenCoder** [comment]
Actaully do you do all of your programming on github? I can teach you how to do it on your local computer if you want, its much easier imo.

File: data/mods/ssb/moves.js (hunk 1 above)
Comment (tmagicturtle) [reply data/mods/ssb/moves.js]: You're right, it isn't needed. Thank you.

**Comment by tmagicturtle** [comment]
I do my testing on my computer but I have no idea how git works so when I upload and do all git-related things, I do use the web interface for github, yes

**Comment by HoeenCoder** [comment]
Its not too hard, I'd be happy to teach you later.

**Comment by HoeenCoder** [comment]
Also did the animations playout when you tested this? Because I'm not seeing any `-anim` messages.

**Comment by tmagicturtle** [comment]
Crap. For some reason I thought I fixed that. I'm sorry.

**Review by HoeenCoder — COMMENTED** [review]
Missing Kings Shield animation, also animations go in onPrepareHit

**Pull request merged**